corehq/apps/cloudcare/static/cloudcare/js/backbone/cases.js

```diff
@@ -173,7 +173,7 @@ hqDefine('cloudcare/js/backbone/cases.js', function () {
             var self = this,
                 text = self.lookupField(col),
                 td = $("<td/>");
-            if (text) {
+            if (text !== null) {
```

Comment:
I think it shouldn't be undefined, and false and blanks would be handled correctly, so i'm going to merge